internal/ansible/events/log_events.go

```diff
@@ -45,6 +46,7 @@ type EventHandler interface {
 
 type loggingEventHandler struct {
 	LogLevel LogLevel
+	mux      *sync.Mutex
```

Comment:
This mutex is never initialized anywhere. Either make `Handle()`'s receiver a pointer and make this a value reference, or set `loggingEventHandler.mux = new(sync.Mutex)` when constructed.